### PR TITLE
SUP-9286 Player Button size not saved on Android and iPad

### PR DESCRIPTION
**Pull request by ErezBouskila**
@OrenMe

### CODE REVIEW

File: modules/KalturaSupport/kalturaIframeClass.php [hunk 1]
```diff
@@ -1328,7 +1328,7 @@ function getIFramePageOutput( ){
 
 <script type="text/javascript">
     var customCSS = <?php echo $customCss ?>;
-    if (['kWidget'] && !window['kWidget'].isMobileDevice() && customCSS){
+    if (['kWidget'] && !window['kWidget'].isMobileDevice() && customCSS || window['kWidget'].isIOS() || window['kWidget'].isAndroid()){
```
Comment (OrenMe) [inline modules/KalturaSupport/kalturaIframeClass.php]: @ErezBouskila need to include test if mobile skin flag is enabled, as mobile device by itself is not sufficient

**Comment by kaltura-hooks** [comment]
Hi @ErezBouskila,
Thank you for contributing this pull request!
Please sign the Kaltura CLA so we can review and merge your contribution.
Learn more at http://bit.ly/KalturaContrib

**Comment by ErezBouskila** [comment]
@OrenMe made the requested change, now if the mobileSkin is enabled it will only add custom CSS for desktop, if it is disabled it will add the CSS for desktop and mobile devices.
Checked and confirmed for all relevant tests.

File: modules/KalturaSupport/kalturaIframeClass.php [hunk 2]
```diff
@@ -1328,7 +1328,9 @@ function getIFramePageOutput( ){
 
 <script type="text/javascript">
     var customCSS = <?php echo $customCss ?>;
-    if (['kWidget'] && !window['kWidget'].isMobileDevice() && customCSS){
+    var skin =  window["kalturaIframePackageData"].playerConfig.layout ? window["kalturaIframePackageData"].playerConfig.layout.skin : "kdark";
+    var mobileSkin = window['kWidget'].isChromeCast() || ( window["kalturaIframePackageData"].playerConfig.vars["EmbedPlayer.EnableMobileSkin"] === true && skin === "kdark" && window['kWidget'].isMobileDevice() && !window['kWidget'].isWindowsPhone() );
```
Comment (OrenMe) [inline modules/KalturaSupport/kalturaIframeClass.php]: @ErezBouskila please add a protection here for window['kWidget'] and for playerConfig.vars

Comment (ErezBouskila) [inline modules/KalturaSupport/kalturaIframeClass.php]: Will add the window, which playerConfig.vars to protect?

Comment (OrenMe) [inline modules/KalturaSupport/kalturaIframeClass.php]: U are accessing nested object so need to check if exist:
* window["kalturaIframePackageData"].playerConfig.layout => check that both window["kalturaIframePackageData"], window["kalturaIframePackageData"].playerConfig, window["kalturaIframePackageData"].playerConfig.vars exist
* window['kWidget'].* => check that window['kWidget'] exist
* Anything else I missed.

**Comment by ErezBouskila** [comment]
@OrenMe

**Pull request merged**